issue2/html_task.js

```diff
@@ -0,0 +1,38 @@
+var webdriver = require('selenium-webdriver');
+var assert = require('chai').assert;
+var { Builder, By } = require("selenium-webdriver");
+
+describe("Testing Scholars Project", function () {
+    this.timeout(5 * 1000 * 60);
+    var driver = new webdriver.Builder()
+        .forBrowser('chrome')
+        .build();
+
+    before(function setupWebdriver(done) {
+        driver.get("file:///C:/Users/oemma/mocha_test/Dufuna-Fem/submissions/desolaOluseye/HTML/productPage.html").then(done)
```

Comment:
We shouldn't have hard-coded file directories since the user might not have these locations on their PC.